### ISSUE TITLE
[BUG] octoprint/octoprint:latest or :1.8.6 not up to date

### ISSUE THREAD

**Issue by fischer-felix** [OctoPrint/octoprint-docker#242]
**Describe the bug**
When deploying octoprint using the official image and the tag `latest` or `1.8.6`, octoprint itself shows the version as being 1.8.1.

**Container Details**

please run `docker inspect --format '{{ index .Config.Labels "org.opencontainers.image.created"}}' octoprint/octoprint:<tag_you_are_using>' and list the date returned.
2022-10-18T09:25:10.308Z

**To Reproduce**
1. Deploy OctoPrint using `octoprint/octoprint:latest` or `octoprint/octoprint:1.8.6`
2. Open the webinterface
3. Go to Settings > About OctoPrint
4. See that Version is 1.8.1

**Expected behavior**
OctoPrint is _actually_ on version 1.8.6 as the tag suggests.

**Screenshots**
![Screenshot_2022 10 30_21:11:18](https://user-images.githubusercontent.com/65448408/198899541-0d8306a0-1755-4ed6-b5ec-35d1f47dab1b.png)
![Screenshot_2022 10 30_21:11:54](https://user-images.githubusercontent.com/65448408/198899562-30dc0c27-852e-4f32-8a65-844178d64384.png)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox
 - Version: 106.0.2

**Comment by Paraphraser** [comment]
I don't know if this will help but I'm not seeing the same pattern.

Same result from the "inspect":

```
$ docker inspect --format '{{ index .Config.Labels "org.opencontainers.image.created"}}' octoprint/octoprint:latest 
2022-10-18T09:25:10.308Z
```

When I search the image for a version:

```
$ docker image inspect octoprint/octoprint:latest | grep "1\.8"
                "octoprint_ref=1.8.6",
```

When I ask the process to tell me its version:

```
$ docker exec octoprint octoprint --version
octoprint, version 1.8.6
```

And when I look in the UI:

<img width="842" alt="Screen Shot 2022-10-31 at 10 15 47" src="https://user-images.githubusercontent.com/34226495/198906730-1a70b970-4349-4fc5-9918-b06add2d3b0a.png">

<img width="712" alt="Screen Shot 2022-10-31 at 10 09 03" src="https://user-images.githubusercontent.com/34226495/198906743-d1879630-b13e-4e06-bf3c-aac9649704c8.png">

**Comment by Paraphraser** [comment]
Hmmm. My compose file has

```
- ./volumes/octoprint:/octoprint
```

so that's probably close to what you are doing, right?

The "rules" I follow (I'm not sure whether they are actual rules or just how I approach it) are that, (1) if I get a prompt to update Octoprint, I wait for a new image to appear on DockerHub; (2) if I get a prompt to update a plug-in, I do that in the plug-in manager (which is persisted).

I learned that first rule the hard way and I'm pretty sure I had to nuke my persistent store and start over to recover.
